text/0000-clippy-uno.md

```diff
@@ -0,0 +1,263 @@
+- Feature Name: clippy_uno
+- Start Date: 2018-06-14
+- RFC PR: (leave this empty)
+- Rust Issue: (leave this empty)
+
+# Summary
+[summary]: #summary
+
+Release Clippy 1.0, in preparation for it being shipped via rustup and eventually available via Rust Stable.
+
+# Motivation
+[motivation]: #motivation
+
+See also: [The Future of Clippy][future]
+
+Clippy, the linter for Rust, has been a nightly-only plugin to Rust for many years.
+In that time, it's grown big, but it's nightly-only nature makes it pretty hard to use.
+
+The eventual plan is to integrate it in Rustup á la Rustfmt/RLS so that you can simply fetch prebuilt binaries
+for your system and `cargo clippy` Just Works ™️. In preparation for this, we'd like to nail down various things
+about its lints and their categorization.
+
+[future]: https://manishearth.github.io/blog/2018/06/05/the-future-of-clippy-the-rust-linter/
+
+# Guide-level explanation
+[guide-level-explanation]: #guide-level-explanation
+
+## Usage and lint philosophy
+
+We expect clippy to be used via `cargo clippy`.
+
+Clippy aims to follow the general Rust style. It may be somewhat opiniated in some situations.
+
+In general clippy is intended to be used with a liberal sprinkling of `#[allow()]` and `#[warn()]`; _it is okay_ to
+disagree with Clippy's choices. This is a weaker philosophy than that behind rustc's lints, where usually flipping
```

Comment:
Is it officially **C**lippy or **c**lippy? I never new, and this RFC is not consistent either.

Reply:
Doesn't seem consistent yet - see line 34 and 35 :)

Reply:
Changed to Clippy.